### PR TITLE
Run testgrid after deploy. Require instances and os specs from the cli

### PR DESCRIPTION
**Pull request by emosbaugh**

### CODE REVIEW

File: kurlkinds/pkg/apis/cluster/v1beta1/installer_types.go [hunk 1]
```diff
@@ -106,7 +106,7 @@ type Kubernetes struct {
 }
 
 type Kurl struct {
-	AdditionalNoProxyAddresses   []string `json:"additionalNoProxyAddresses,omitempty" yaml:"proxyAddress,omitempty"`
```
Comment (laverya) [inline kurlkinds/pkg/apis/cluster/v1beta1/installer_types.go:109]: was this intended?

File: testgrid/specs/latest.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+- name: ""
```
Comment (laverya) [inline testgrid/specs/latest.yaml:1]: can we also include the spec for longhorn? https://kurl.sh/longhorn

File: kurlkinds/pkg/apis/cluster/v1beta1/installer_types.go (hunk 1 above)
Comment (emosbaugh) [reply kurlkinds/pkg/apis/cluster/v1beta1/installer_types.go:109]: Isn't that a bug. It's the wrong yaml tag. Or am I misunderstanding your comment?

File: testgrid/specs/latest.yaml (hunk 2 above)
Comment (emosbaugh) [reply testgrid/specs/latest.yaml:1]: I'll add

File: testgrid/tgrun/pkg/scheduler/addons.go [hunk 3]
```diff
@@ -34,6 +34,9 @@ func generateAllAddOns() ([]kurlv1beta1.InstallerSpec, error) {
 	return installerSpecs, nil
 }
 
+// hack
+var previouslyGeneratedNames = []string{}
```
Comment (laverya) [inline testgrid/tgrun/pkg/scheduler/addons.go:38]: what's this used for? (I realize it's not new)

**Review by laverya — APPROVED** [review]

File: testgrid/tgrun/pkg/scheduler/addons.go (hunk 3 above)
Comment (emosbaugh) [reply testgrid/tgrun/pkg/scheduler/addons.go:38]: i have no idea. i moved it closer to where its referenced

File: kurlkinds/pkg/apis/cluster/v1beta1/installer_types.go (hunk 1 above)
Comment (laverya) [reply kurlkinds/pkg/apis/cluster/v1beta1/installer_types.go:109]: I misread the +- 🤦

**Review by laverya — APPROVED** [review]

**Pull request merged**